aws-databricks/main.tf

```diff
@@ -0,0 +1,105 @@
+terraform {
+  required_version = "~> 1.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+    databricks = {
+      source  = "databrickslabs/databricks"
+      version = "0.4.6"
+    }
+    null = {
+      source  = "hashicorp/null"
+      version = "3.1.0"
```

Comment:
Why the difference in versioning strategies? We seem to be combining sem (~>) and pinning?